go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/unidoc/unichart
 
 go 1.16
 
-require github.com/stretchr/testify v1.7.1
+require (
+	github.com/stretchr/testify v1.7.1
+	github.com/unidoc/unipdf/v3 v3.49.0
```

Comment:
can we put this into go.mod under the examples/ only?  seems like unnecessary dependency for unichart itself?

Reply:
I've separate the examples folder into it's own module in the latest updates